master-nonlatent/master.cfg

```diff
@@ -114,7 +114,13 @@ c["workers"].append(aix_worker)
 
 # Docker Library
 dockerlibrary_worker = mkWorker(
-    os.getenv("MASTER_NONLATENT_DOCKERLIBRARY_WORKER", default="bb-rhel8-docker"), properties={"jobs": 1, "save_packages": True}
+    os.getenv("MASTER_NONLATENT_DOCKERLIBRARY_WORKER", default="bb-rhel8-docker"),
+    properties={
+        "jobs": 1,
+        "save_packages": os.getenv("ENVIRON") == "PROD",
```

Comment:
Isn't it useful to save packages for testing purposes for dev?

Reply:
the `dockerlibrary_worker` doesn't use the property anyway. The outcome is quay.io containers which I've handled in the *-manifest.sh script anyway.

The outcome is tested on the wordpress test.

I haven't yet seen a case where pushing to different quay.io tag or saving the container and manifest as a tarball would be useful. Might change later, but for now I think its ok.